agent/agent-filtering.md

```diff
@@ -0,0 +1,39 @@
+# Filter interface
+
+The filtering capability allows running an arbitrary filter at the entry point of the application.
+The instrumentation points use the result of the evaluation to block the execution.
```

Comment:
I am not 100% this clarifies whether the span is going to be processed or not or annotated with some sort of error.

Reply:
this is a good point. At least in Java the span lifecycle is orthogonal to filter processing. The filters can add any metadata that they want.